apps/server/src/toast-participants/toast-participants.controller.ts

```diff
@@ -10,4 +11,21 @@ export class ToastParticipantsController {
   findAll() {
     return this.toastParticipantsService.findAll();
   }
+
+  @Get('allUsers/:toastId')
+  findAllParticipantsForToastId(@Param('toastId') toastId: string) {
+    return this.toastParticipantsService.findAllParticipantsForToastId(toastId);
+  }
+  @Post()
+  createToastParticipant(
+    @Body() newToastParticipant: CreateToastParticipantDto
```

Comment:
I am not sure this covers a case of multiple participants at once. should probably add that